src/components/TextField/OfficeTextField.vue

```diff
@@ -56,20 +64,51 @@
         @Model("input", {type: String}) private text!: string;
 
         @Prop({type: Boolean, default: false}) private disabled!: boolean;
+        @Prop({type: Boolean, default: false}) private autoAdjustHeight!: boolean;
         @Prop({type: Boolean, default: false}) private multiline!: boolean;
         @Prop({type: Object, default: null}) private iconProps!: IOfficeIconStyleProps;
         @Prop({type: Boolean, default: false}) private borderless!: boolean;
         @Prop({type: String, default: ""}) private inputClassName!: string;
         @Prop({type: String, default: null}) private iconClass!: string;
         @Prop({type: String, default: ""}) private label!: string;
+        @Prop({type: String, default: undefined}) private prefix!: string;
+        @Prop({type: String, default: undefined}) private suffix!: string;
         @Prop({type: String, default: null}) private errorMessage!: string;
         @Prop({type: Boolean, default: false}) private required!: boolean;
         @Prop({type: Boolean, default: false}) private resizable!: boolean;
         @Prop({type: Boolean, default: false}) private underlined!: boolean;
+        @Prop({
+            type: () => {
+                /*
+                    noop
+                */
+            },
+            default: null
+        }) private onChange!: (event: Event, value: string) => void;
+        @Prop({
+            type: () => {
+                /*
+                   noop
+               */
+            },
+            default: null
+        }) private onFocus!: (event: Event) => void;
+        @Prop({
+            type: () => {
+                /*
+                    noop
+                */
+            },
+            default: null
+        }) private onBlur!: (event: Event) => void;
+
 
         private focused: boolean = false;
         private id: number = (Math.random() * 100000) + 1;
 
+        public created() {
```

Comment:
Do this on the `mounted` lifecycle hook instead on `created`, otherwise the "TextField" is not yet mounted to the DOM, so the adjustHeight doesn't make much sense.